src/pdf_beamtime_interfaces/srv/BlueskyInterruptMsg.srv

```diff
@@ -0,0 +1,3 @@
+string name
```

Comment:
If the structure doesn't matter, I would send an int and receive booleans.

Reply:
Changed the name to 'interrupt_type'. Now it returns a boolean. 
https://github.com/maffettone/erobs/blob/b7595c9042764872d4c9c7d38f3960a33446b96c/src/pdf_beamtime_interfaces/srv/BlueskyInterruptMsg.srv#L3